shared/tsconfig.strict.json

```diff
@@ -1,9 +1,10 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
+    "assumeChangesOnlyAffectDirectDependencies": false,
```

Comment:
I don't know the status of the `noImplicitAny` project, but assume we don't want this here